.github/workflows/cmake.yml

```diff
@@ -250,7 +260,7 @@ jobs:
           - name: Ubuntu 16.04 Clang
             os: ubuntu-16.04
             compiler: clang-6.0
-            # note: github preinstalls compilers now, see https://github.com/actions/virtual-environments/pull/369/
+            packages: clang-6.0
```

Comment:
I have separated out this change because it is affecting my other PRs.
https://github.com/zlib-ng/zlib-ng/pull/926/files

Reply:
OK! I'll rebase when your PR is merged.